pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -678,6 +678,11 @@ func (ctrl *Controller) syncImageConfig(key string) error {
 		}
 	}
 
+	// append the internalRegistryHostname to allowedRegistries list
```

Comment:
(Absolutely non-blocking I don’t think this comment adds any value right now; the code says just that already. A comment saying _why_ we add it could be useful.)